### DISCUSSION TITLE
Any howto on updating the dbx file?

### DISCUSSION THREAD

**Discussion by MichaelTrip** [tianocore/edk2#3276] [Q&A] [answered]
As mentioned in a previous topic, there is a problem wih a Windows Update that doesn't want to be installed. A DBX file needs to be updated / installed. I tried this manual but it doesn´t mention any form of updating the DBX file.

Can anyone point me in the right direction?

**Comment by spbrogan** [comment, marked as answer]
If you are using Windows there are powershell scripts that can help you make and install different payloads. 
Hopefully you have an understanding of the various variables defined in UEFI spec, the signing requirements, and how your system is configured.  

Powershell scripts:
https://docs.microsoft.com/en-us/powershell/module/secureboot/?view=windowsserver2022-ps

Basic docs about secure boot:
https://docs.microsoft.com/en-us/windows-hardware/design/device-experiences/oem-secure-boot
